client/src/db.json

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
db.json should be located in the root folder. This is not needed

Reply:
Oh, that was still a "left-over" from when I had it installed in the client 🙈